client/deposits/instant-deposits/modal.js

```diff
@@ -25,7 +25,7 @@ const InstantDepositModal = ( {
 	const description = createInterpolateElement(
 		/* translators: <a> - instant payout doc URL */
 		__(
-			'Need cash in a hurry? Instant deposits are available within 30 minutes for a nominal 1% service fee. <a>Learn more</a>',
+			'Need cash in a hurry? Instant deposits are available within 30 minutes for a nominal 1.5% service fee. <a>Learn more</a>',
```

Comment:
As I suggested in the issue, what do you think about using `100*amount/fee` (maybe rounded to 2 decimal points to prevent floating-point shenanigans) to calculate the fee? It will work just fine as long as we keep the fee a simple percentage.

Reply:
I believe this would work in most instances, but it leaves a point of discrepancy due to rounding. If it's rounded to 1 decimal place, it _should_ always work. I feel like a more solid solution would be to pass the value from the server since it would just be one extra line in the server code.